packages/sources/bea/test/e2e/price.test.ts

```diff
@@ -0,0 +1,43 @@
+import { assertSuccess } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../../src/adapter'
+
+describe('execute', () => {
+  const jobID = '1'
+  const execute = makeExecute()
+  process.env.API_KEY = process.env.API_KEY ?? 'test_api_key'
+
+  describe('successful requests', () => {
+    const requests = [
+      {
+        name: 'id not supplied',
+        testData: {
+          data: {},
+        },
+      },
+      {
+        name: 'last not supplied',
+        testData: {
+          data: {
+            series: 'DPCERG',
+          },
+        },
+      },
+      {
+        name: 'serie not supplied',
+        testData: {
+          data: {
+            last: 4,
+          },
+        },
+      },
+    ]
+
+    requests.forEach((req) => {
+      it(`${req.name}`, async () => {
+        const adapterResponse = await execute(req.testData as AdapterRequest)
+        assertSuccess(adapterResponse.statusCode, adapterResponse, jobID)
+      })
+    })
+  })
+})
```

Comment:
Could we please add integration tests with a mock endpoint as well?

That would give a good opportunity to test that the sorting works as expected as well, by purposely giving the response items out of order.